docs/Custom_Device_Types.md

```diff
@@ -104,19 +104,24 @@ The *Close* case executes after the PCL has finished executing. You should close
 
 ### Inline-Async Hardware Interface
 
-By checking the **Use Inline-Async API** for the *Inline HW Interface* Custom Device Execution Mode in the niveristand-custom-device-wizard, an inline-async custom device template will be generated.
+You can generate an inline-async custom device template from the niveristand-custom-device-wizard. When creating a project, set the execution mode to **Inline HW Interface** and enable **Use Inline-Async API**.
 
 ![](images/Use_Inline_Async_API.jpg)
 
-By using the [Inline-Async-API](https://github.com/ni/niveristand-custom-device-development-tools/tree/main/inline-async-api) the inline-async template framework provides initializing/launching/cleanup of the asynchronous VIs, error handling/reporting, and data transfer between inline and asynchronous VIs.
+By using the [Inline-Async-API](https://github.com/ni/niveristand-custom-device-development-tools/tree/main/inline-async-api), the inline-async template framework can perform the following actions.
+* Initializing asynchronous VIs.
+* Launching asynchronous VIs.
+* Cleaning up asynchronous VIs.
+* Handling and reporting errors.
+* Transferring data between inline and asynchronous VIs.
 
-The RT Driver VI of an inline custom device can communicate channel data with VeriStand while launching an asynchronous loop(s) to handle nondeterministic operations.
+The RT Driver VI of an inline custom device can communicate channel data with VeriStand. While doing so, the VI can also launch an asynchronous loop(s) to handle nondeterministic operations.
 
 One example of a nondeterministic operation is writing data to a log file. The RT Driver VI of the inline custom device communicates with the asynchronous loop(s) using RT FIFOs.
 
 ![](images/Inline-Async_Engine_lvlib_RT_Driver.jpg)
 
-**Note:** Data must be consumed from the RT FIFOs at a fast enough rate or the mechanism will overflow.
+**Note:** The custom device must consume data from the RT FIFOs at a fast enough rate or it will overflow.
```

Comment:
@phutanu I assumed we are talking about the custom device. Let me know if we meant something else.

Reply:
We are talking about the communication between the inline and the async components of the custom device. This communication can go in any direction between these two. You can have multiple async processes that can read from the inline side, and also status data that can be read from the async loops.

Reply:
@phutanu How does this sound then? 

"An inline-async custom device must internally communicate RT FIFO data at a fast enough rate to prevent overflow."

Reply:
@WillCawley for me is not clear where the data goes/comes. I think we should not let the users guess, and just say that they should be mindful on the RT FIFO communication taking place between the inline and async VIs as to not loose data.

Reply:
How about this?
"While the VI is using RT FIFOs, data may be lost if the custom device does not run at a fast enough rate."

Reply:
The communication can go both ways. So both inline and async VIs need to be aware of RT FIFOs read/write.

What do you think about this:

"While the RT Driver VI is using RT FIFOs, data may be lost if elements are not read at a fast enough rate."